public/index.html

```diff
@@ -11,6 +11,17 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <!--
+    <!--
```

Comment:
We should get rid of one of the two `<!--`s above and add one before the line that starts with `Notice the use of` below.

This section of HTML should look like this:

```html
    <!--
      The Open Graph protocol allows links to be expanded by Twitter, Facebook, and other platforms.
      The variables specified here (and in title) are located in .env
    -->
    <meta property="og:site_name" content="%REACT_APP_SITE_NAME%" />
    <meta property="og:type" content="object" />
    <meta property="og:title" content="%REACT_APP_TITLE%" />
    <meta property="og:description" content="%REACT_APP_DESCRIPTION%" />
    <meta property="og:url" content="%REACT_APP_URL%" />

    <!--
      Notice the use of %PUBLIC_URL% in the tags above.
      It will be replaced with the URL of the `public` folder during the build.
```